pro/src/core/Venue/adapters/__specs__/useGetVenue.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { renderHook } from '@testing-library/react-hooks'
+
+import { api } from 'apiClient/api'
+import { GetVenueResponseModel } from 'apiClient/v1'
+
+import { useGetVenue } from '../getVenueAdapter'
+
+describe('useGetVenue', () => {
+  it('should return loading payload then success payload', async () => {
+    const apiVenue: GetVenueResponseModel = {
+      publicName: 'Cinéma des iles',
+    } as GetVenueResponseModel
```

Comment:
Voila les champs obligatoire de la reponse:
 ```js
 collectiveDomains: Array<GetVenueDomainResponseModel>;
  dateCreated: string;
  fieldsUpdated: Array<string>;
  id: string;
  isValidated: boolean;
  isVirtual: boolean;
  managingOfferer: GetVenueManagingOffererResponseModel;
  managingOffererId: string;
  name: string;
  nonHumanizedId: number;
  ```
  
  Ca donnerai : 
  ```suggestion
  const apiVenue: GetVenueResponseModel = {
    publicName: 'Cinéma des iles',
    collectiveDomains: [],
    dateCreated: '2022-07-29T12:18:43.087097Z',
    fieldsUpdated: [],
    id: 'AE',
    isValidated: true,
    isVirtual: false,
    managingOfferer: {
      city: 'Paris',
      dateCreated: '2022-07-29T12:18:43.087097Z',
      fieldsUpdated: [],
      id: 'AA',
      isValidated: true,
      name: 'Test structure name',
      postalCode: '75001',
    },
    managingOffererId: 'AA',
    name: 'Lieu name',
    nonHumanizedId: 12,
} 
```